src/tests/ftest/util/general_utils.py

```diff
@@ -666,8 +666,57 @@ def convert_list(value, separator=","):
     return separator.join([str(item) for item in value])
 
 
-def stop_processes(hosts, pattern, verbose=True, timeout=60, added_filter=None,
-                   dump_ult_stacks=False):
+def dump_engines_stacks(hosts, verbose=True, timeout=60, added_filter=None):
+    """Signal the engines on each hosts tog their ULT stacks dump.
```

Comment:
Seems to have spelling issue
```suggestion
    """Signal the engines on each hosts to generate their ULT stacks dump.
```

Reply:
Ok, done in next commit